tests/benchmark/test_worst_compute.py

```diff
@@ -2255,3 +2255,63 @@ def test_worst_push(
         post={},
         tx=tx,
     )
+
+@pytest.mark.parametrize(
+    "opcode",
+    [Op.RETURN, Op.REVERT],
+)
+@pytest.mark.parametrize(
+    "return_size, return_non_zero_data",
+    [
+        pytest.param(0, False, id="empty"),
+        pytest.param(1024, True, id="1KiB of non-zero data"),
+        pytest.param(1024, False, id="1KiB of zero data"),
+        pytest.param(1024 * 1024, True, id="1MiB of non-zero data"),
+        pytest.param(1024 * 1024, False, id="1MiB of zero data"),
+    ],
+)
+def test_worst_return_revert(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    fork: Fork,
+    opcode : Op,
```

Comment:
```suggestion
    opcode: Op,
```
I think this is the lint issue.